shortstories/tests.py

```diff
@@ -71,3 +71,19 @@ def test_minute_count_function_longtext(self):
         current_shortstory.minute_to_read = minute_count(current_shortstory.body)
         self.assertTrue(current_shortstory.minute_to_read==1)
+
+class ShortStoryFormTests(TestCase):
+    def test_ShortStory_invalid(self):
+        user_id = User.objects.create(id=1)
+        form = ShortStoryForm(data={'author':'', 'status':'', 'title':"", 'body':"", 'publication_date':''})
+        self.assertFalse(form.is_valid())
+
+    def test_ShortStory_valid(self):
+        user_id = User.objects.create(id=1)
```

Comment:
rename this one and the object in the above test to user.

Reply:
done